roles/openstack-stack/templates/heat_stack.yaml.j2

```diff
@@ -73,7 +73,92 @@ outputs:
 
 resources:
 
+  api_lb:
+    type: OS::Neutron::LBaaS::LoadBalancer
+    properties:
+      name:
+        str_replace:
+          template: openshift-ansible-cluster_id-api-lb
+          params:
+            cluster_id: {{ stack_name }}
+      vip_address: {{ api_lbaas_ip }}
+      vip_subnet: { get_resource: service_subnet }
```

Comment:
This needs to be in a `use_kuryr` block, too, otherwise it won't find the `service_subnet` I think. Or can we make this work with the non-kuryr deployments as well by using the `subnet` subnet when `use_kuryr` is `false`?

Reply:
Also, this uses the service_subnet's IP range. What's the plan for a publicly accessible LB? Will it be a separate resource with a floating IP?